main.tf

```diff
@@ -25,13 +25,10 @@ resource "azurerm_cdn_endpoint" "cdn_endpoint" {
   profile_name                  = local.cdn_endpoint[each.key].profile_name
   is_http_allowed               = local.cdn_endpoint[each.key].is_http_allowed
   is_https_allowed              = local.cdn_endpoint[each.key].is_https_allowed
-  content_types_to_compress     = local.cdn_endpoint[each.key].content_types_to_compress
```

Comment:
why removing [content_types_to_compress](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_endpoint#content_types_to_compress-1) it still exists int the latest version auf azurerm 4